packages/backend/src/core/EmailService.ts

```diff
@@ -239,6 +252,12 @@ export class EmailService {
 			related_domains: string[];
 		};
 
+		if (json.message) {
```

Comment:
Is 'every' message indicates 'api key' error?

Reply:
When the api usage exceeds the limit, will get `{ message: "error message content" }`, returned with only the `message` attribute. I've reworked the judgment method